.pipelines/stages/jobs/nuget-packaging-job.yml

```diff
@@ -79,14 +81,40 @@ jobs:
     value: 'Microsoft.AI.DirectML.1.14.2.zip'
   - name: dml_url
     value: "https://www.nuget.org/api/v2/package/Microsoft.AI.DirectML/1.14.2"
+  
+  - name: ort_arm64_dir
+    value: 'onnxruntime-win-arm64-1.17.3'
+  - name: ort_arm64_zip
+    value: 'onnxruntime-win-arm64-1.17.3.zip'
+  - name: ort_arm64_url
+    value: "https://github.com/microsoft/onnxruntime/releases/download/v1.17.3/onnxruntime-win-arm64-1.17.3.zip"
+
   workspace:
     clean: all
   steps:
-  - task: UsePythonVersion@0
-    inputs:
-      versionSpec: 3.10
-      addToPath: true
-      architecture: $(arch)
+  - script: |
+      MKDIR $(Agent.ToolsDirectory)\Python\3.11.0\arm64
```

Comment:
Could you please put them in the VM's image?

Reply:
Yeah, will move this part in image later

Reply:
Base  on  Adrian's suggestion, it would be better to create a new image and new agent pool for new project. I would like to know whether ort-genai project can reuse the pool and image which are used by ORT build and release. If yes, then I think just modify the image config and rebuild image is enough. If not, then it will take more effort and time  to create new image and pool.

Reply:
personally, I wish ORT eco-system team can use their own pool and image as possible.
Because there're higher code standards, performance and stability requirements.
And we'll give a high priority on ORT pipelines. @snnn

Reply:
It's fine if you want to create a new image and new agent pool . But since you need to use QNN, you need to get help from the QNN EP team. Otherwise you would not be able to get QNN SDK at all.

Reply:
in order to avoid breaking build and release process of ORT since they are build so often and release daily. I have set up new image and agent pool for ort-genai to publish windows arm64 pkgs